GM/gmpart.py

```diff
@@ -225,6 +225,7 @@ async def messages_list(self, aiogoogle, user_creds, messages_num=5):
         for message in messages_ids['messages']:
             raw_messages.append(
                 self.get_gmail_message(
+                    aiogoogle= aiogoogle,
```

Comment:
It causes `got multiple values for argument 'aiogoogle'`
All in all, the decorator you just moved, which also uses in functions `get_gmail_message()` and `authorize_uri()` do add this kv argument. When `aiogoogle` is creating - it opens a new session, so a new session in any way will be created when `get_gmail_message()` is called (maybe).